src/sys/windows/selector.rs

```diff
@@ -464,16 +481,28 @@ impl SelectorInner {
 
     unsafe fn update_sockets_events(&self) -> io::Result<()> {
         let mut update_queue = self.update_queue.lock().unwrap();
-        loop {
-            let sock = match update_queue.pop_front() {
-                Some(sock) => sock,
-                None => break,
-            };
+        let mut update_err: bool = false;
+
```

Comment:
Extra line snuck in

Reply:
removed, btw I like many extra lines :).